wrangler-transform/src/main/java/co/cask/wrangler/Wrangler.java

```diff
@@ -233,6 +239,38 @@ public void configurePipeline(PipelineConfigurer configurer) throws IllegalArgum
     }
   }
 
+  @Override
+  public void prepareRun(StageSubmitterContext context) throws Exception {
+    super.prepareRun(context);
+    List<String> inputFields = new ArrayList<>();
+    List<String> outputFields = new ArrayList<>();
+    Schema inputSchema = context.getInputSchema();
+    if (checkSchema(inputSchema, "input")) {
+      //noinspection ConstantConditions
+      inputFields = inputSchema.getFields().stream().map(Schema.Field::getName).collect(Collectors.toList());
+    }
+    Schema outputSchema = context.getOutputSchema();
+    if (checkSchema(outputSchema, "output")) {
+      //noinspection ConstantConditions
+      outputFields = outputSchema.getFields().stream().map(Schema.Field::getName).collect(Collectors.toList());
+    }
+    FieldOperation dataPrepOperation = new FieldTransformOperation("DataPrep", config.directives, inputFields,
```

Comment:
The other names seem to be verbs, whereas this is a noun. I would call it 'Prepare' or 'Prepare Data' (are spaces allowed?)